sparql-ns.ttl

```diff
@@ -0,0 +1,403 @@
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX xsd:     <http://www.w3.org/2001/XMLSchema#>
+PREFIX sparql:  <http://www.w3.org/ns/sparql#>
+PREFIX dc:      <http://purl.org/dc/terms/>
+PREFIX owl:     <http://www.w3.org/2002/07/owl#>
+
+<http://www.w3.org/ns/sparql#> a owl:Ontology ;
+     dc:date "2025-06-30" ;
+     dc:description "Terms used by SPARQL 1.2" ;
+     owl:versionInfo ""
+     .
+
+##
+
+sparql:Function rdf:type rdfs:CLass .
+sparql:Operator rdf:type rdfs:CLass .
+sparql:FunctionalForm rdf:type rdfs:CLass .
+
+## @@ToDo
+
+## Operators
+
+# Arithmetic: +, -, *, /
+# Comparision: = , !=, >, >=, <, <=
+# Unary: !, +, -
+# Boolean: && and ||
+# Aggregates:          COUNT, SUM, MIN, MAX, AVG, SAMPLE, GROUP_CONCAT
+# Aggregates distinct: COUNT, SUM, MIN, MAX, AVG, SAMPLE, GROUP_CONCAT
+# Check WRT BuiltInCall
+
+##@@
+
+## Operators
+
+sparql:plus rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:subtract rdf:type sparql:Operator
+    dc:description "" ;
+    rdfs:isDefinedBy <http://www.w3.org/TR/sparql12-query/> ;
+    .
+
+sparql:multiple rdf:type sparql:Operator
+    dc:description "" ;
```

Comment:
This operator multiplies two numeric expressions and returns the product.

Reply:
```suggestion
    dc:description "This operator multiplies two numeric expressions and returns the product." ;
```